scripts/createComponent.js

```diff
@@ -98,12 +98,14 @@ const componentFactory = (directory) => {
 const demoHtmlFactory = (directory) => {
   const template = `${copyrightBannerHtml}
 <!DOCTYPE html>
-<html id="theme">
+<html lang="en-US" id="theme">
   <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>${componentName} | iTwinUI</title>
     <link
       rel="icon"
-      href="../assets/favicon.ico"
+      href="../assets/favicon.svg"  type="image/svg+xml"
     />
```

Comment:
forgot to add apple touch here too

Reply:
Apple Touch Icon and Open Graph meta data added to demo pages & Create Component JS.